src/components/lib/VDropdown/VDropdown.vue

```diff
@@ -179,6 +179,18 @@ export default {
       toggle();
     }
 
+    function clickListener(e) {
+      isOpen.value = false;
+    }
+
+    onMounted(() => {
```

Comment:
This logic isn't specific to the dropdown component and may be helpful in other contexts. We can potentially separate it out as a reusable directive or composition function. 

Here's [one possible implementation](https://thewebdev.info/2021/05/17/how-to-detect-clicks-outside-an-element-with-vue-js/) as a directive and one [as a composition function](https://vueuse.org/core/onclickoutside/).

We don't have to make this change at the moment — just something to keep in mind for later.